### ISSUE TITLE
keyboard navigation focus is missing after Remove Wall is pressed

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/QA#119]
<b>Test Device</b>

MacBook Air

<b>Operating System</b>

10.13.4

<b>Browser</b>

Safari 11.1

<b>Problem Description</b>

I used VoiceOver to guide me to the "Remove Wall" button. The "Remove Wall" button initially had keyboard navigation focus, i.e. the purple outline. When the "Remove Wall" button was pressed, the keyboard navigation focus disappeared.

This does not occur when I navigate to the "Remove Wall" button using normal keyboard navigation.

<b>Steps to Reproduce</b>

1. Use VoiceOver to guide you to the "Remove Wall" button.
2. Notice the keyboard navigation focus on the "Remove Wall" button.
3. Press the "Remove Wall" button using VoiceOver.
4. Notice the keyboard navigation focus disappear from the "Remove Wall / Add Wall" button.

<b>Visuals</b>

![1](https://user-images.githubusercontent.com/31285590/40069928-ded3d3fe-5829-11e8-9d7e-352d6d4b46d3.png)
![2](https://user-images.githubusercontent.com/31285590/40069936-e3b4696a-5829-11e8-80ac-dbad74eedf80.png)

<details>
<summary><b>Troubleshooting Information</b></summary>

Name: ‪Balloons and Static Electricity‬
URL: https://bayes.colorado.edu/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_all_phet_debug.html?accessibility
Version: 1.4.0-dev.21 2018-04-04 18:25:56 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1 Safari/605.1.15
Language: en-US
Window: 1440x736
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (2.1 INTEL-10.32.48)
GLSL: WebGL GLSL ES 1.0 (1.20)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 16)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"master"},"axon":{"sha":"ceb3cc00","branch":"master"},"balloons-and-static-electricity":{"sha":"f4f3cc64","branch":"master"},"brand":{"sha":"89d28f63","branch":"master"},"chipper":{"sha":"bc7c5d1f","branch":"master"},"dot":{"sha":"e5489d18","branch":"master"},"joist":{"sha":"2cf256a9","branch":"master"},"kite":{"sha":"b6071478","branch":"master"},"phet-core":{"sha":"d8804eee","branch":"master"},"phet-io":{"sha":"4bbddfaa","branch":"master"},"phet-io-website":{"sha":"b6c2b0f5","branch":"master"},"phet-io-wrapper-classroom-activity":{"sha":"1ecd81a1","branch":"master"},"phet-io-wrapper-lab-book":{"sha":"b1c5562c","branch":"master"},"phet-io-wrappers":{"sha":"025cc23d","branch":"master"},"phetcommon":{"sha":"de004c6e","branch":"master"},"query-string-machine":{"sha":"c379f1db","branch":"master"},"scenery":{"sha":"4a4afee2","branch":"master"},"scenery-phet":{"sha":"e68ebc03","branch":"master"},"sherpa":{"sha":"bb14a03a","branch":"master"},"sun":{"sha":"8a187c9f","branch":"master"},"tandem":{"sha":"f8147666","branch":"master"},"vibe":{"sha":"2dc20be7","branch":"master"}}

</details>

**Comment by lmulhall-phet** [comment]
The same thing happens with the "Two-Balloon Experiments" checkbox and the "Reset Balloons" button.

**Comment by lmulhall-phet** [comment]
Happens with the "Reset All" button, but it does not occur with the keyboard navigation icon or the PhET menu button.

**Comment by lmulhall-phet** [comment]
Testing on dev version.

**Comment by jessegreenberg** [comment]
I see, thanks. I just tried on BOSE with VO Safari 11.0.3 I am not able to reproduce on master. Ill try the dev version.

**Comment by lmulhall-phet** [comment]
OK, thanks. Also, not sure why I put this in the QA repo.

**Comment by jessegreenberg** [comment]
Ah, didn't even notice!

**Comment by lmulhall-phet** [comment]
I'm going to move this over to the correct repo and label this one a duplicate.
